index.js

```diff
@@ -31,7 +36,7 @@ function ExplicitInstalls (cb) {
 
 ExplicitInstalls.getPackages = function () {
   return new Promise(function (resolve, reject) {
-    ExplicitInstalls.fs.readFile(path.resolve(__dirname, './packages.json'), 'utf-8', function (err, packages) {
+    ExplicitInstalls.fs.readFile(path.resolve(configDirectory, './packages.json'), 'utf-8', function (err, packages) {
```

Comment:
The public methods `getPackages()` and `getLogos()` will now no longer work independently from the main export.

It seems they should either be made private functions (which will break mocking in tests) or each method should use `process.env.NEI_CONFIG_DIRECTORY || __dirname` on its own.